masakari-controller/controller/masakari_util.py

```diff
@@ -697,104 +838,55 @@ def do_instance_evacuate(self, uuid, targethost):
         :targethost: The name or ID of the host where the server is evacuated.
         """
         try:
-            self.rc_util.syslogout('Call Evacuate API with %s to %s' %
-                                   (uuid, targethost), syslog.LOG_INFO)
+            msg = ("BEGIN do_instance_evacuate: " \
+                   "parameters (uuid=%s, targethost=%s)" \
+                   % (uuid, targethost))
+            LOG.debug(self.rc_util.msg_with_thread_id(msg))
+
+            msg = ('Call Evacuate API with %s to %s' %
+                                   (uuid, targethost))
+            LOG.info(self.rc_util.msg_with_thread_id(msg))
             self.nova_client.servers.evacuate(uuid, host=targethost,
                                               on_shared_storage=True)
+            msg = "END do_instance_evacuate"
+            LOG.debug(self.rc_util.msg_with_thread_id(msg))
+
         except exceptions.ClientException as e:
-            error_code = "[RecoveryControllerUtilApi_0007]"
             msg = ('Fails to call Instance Evacuate API onto %s: %s'
                    % (targethost, e))
-            self.rc_util.syslogout(error_code + msg, syslog.LOG_ERR)
+            LOG.error(self.rc_util.msg_with_thread_id(msg))
             raise
 
-
 class RecoveryControllerUtil(object):
 
     """
     Other utility classes for VM recovery control
     """
 
     def __init__(self, config_object):
+        msg = ("BEGIN __init__: " \
+               "parameters (config_object=%s)" % (config_object))
+        LOG.debug(self.msg_with_thread_id(msg))
+
         self.rc_config = config_object
 
-    def syslogout_ex(self, msgid, logOutLevel):
-        """
-        I output the log to a given log file
-        :msgid : Log output message ID(Monitoring message)
-        :logOutLevel: Log output level
-        """
-        monitoring_message = str(threading.current_thread())\
-            + " --MonitoringMessage--ID:[%s]" % (msgid)
-        self.syslogout(monitoring_message, logOutLevel)
+        msg = "END __init__"
+        LOG.debug(self.msg_with_thread_id(msg))
 
-    def syslogout(self, rawmsg, logOutLevel):
+    def msg_with_thread_id(self, rawmsg):
         """
-        I output the log to a given log file
-        :msg : Log output messages
-        :logOutLevel: Log output level
+        Add thread id to rawmsg.
+        :rawmsg : message.
+        :return : message with thread id.
         """
+        logmsg = ("BEGIN msg_with_thread_id: " \
+                  "parameters (rawmsg=%s)" % (rawmsg))
+        LOG.debug(str(threading.current_thread()) + " " + logmsg)
```

Comment:
IMO, this method shouldn't have BEGIN foo/ END foo message. When you set log level to debug, following messages will be output. I think it's really annoying.

BEGIN msg_with_thread_id: parameters( rawmsg= message_passed_from_caller )
thread_id message_passed_from_caller
END msg_with_thread_id: return message_passed_from_caller